lexer/tiger.lex

```diff
@@ -1,18 +1,125 @@
-type pos = int
+type pos = int;
 type lexresult = Tokens.token
 
 val lineNum = ErrorMsg.lineNum
 val linePos = ErrorMsg.linePos
-fun err(p1,p2) = ErrorMsg.error p1
 
-fun eof() = let val pos = hd(!linePos) in Tokens.EOF(pos,pos) end
+val partialString = ref ""
+val stringStartPos = ref ~1
 
+val commentCount = ref 0
 
-%% 
+fun getEscapedChar c =
+	case c of
+		  "\\^@" => Char.toString(chr(0))
```

Comment:
I was going by the appendix in the textbook. It's less comprehensive than the Tiger manual, but I think it's the resource we should use for this type of thing. I'm assuming Dr. Hilton will base the correctness off the textbook's interpretation of Tiger rather than the manual's. Either way, we can list the discrepancy in the README, and we should reasonably get full credit.
![capture](https://user-images.githubusercontent.com/35435859/51721917-ab368f80-2021-11e9-8171-766c957a01b1.PNG)